README.md

```diff
@@ -10,18 +36,23 @@ Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The app w
 
 Run `ng generate component component-name` to generate a new component. You can also use `ng generate directive|pipe|service|class|guard|interface|enum|module`.
 
-## Build
+## build
 
 Run `ng build` to build the project. The build artifacts will be stored in the `dist/` directory. Use the `--prod` flag for a production build.
 
 ## Running unit tests
 
 Run `ng test` to execute the unit tests via [Karma](https://karma-runner.github.io).
 
-## Running end-to-end tests
-
+**Execute e2e tests:**
 Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protractortest.org/).
+```SH
+$ npm run protractor
```

Comment:
`npm run e2e `instead of `npm run protractor`

`protractor` is not any command listed in `package.json`

Reply:
changed to same